### PR TITLE
fix(styles): reworked get-function for tokens

### PR DESCRIPTION
**Pull request by schaertim**

### CODE REVIEW

File: packages/documentation/.storybook/styles/components/typo.scss [hunk 1]
```diff
@@ -30,7 +30,7 @@ body {
     }
 
     .docs-story > :first-child {
-      font-family: tokens.get('post-body', 'font-family');
+      font-family: tokens.get('post-body-font-family');
```
Comment (alizedebray) [inline packages/documentation/.storybook/styles/components/typo.scss:33]: This PR would be a good opportunity to remove the "post-" prefix, don't you think?
```suggestion
      font-family: tokens.get('body-font-family');
```

**Review by alizedebray — APPROVED** [review]

**Comment by alizedebray** [comment]
Please notify all developers as soon as this PR is merged so that they can update their PR's.

**Review by gfellerph — CHANGES_REQUESTED** [review]
@schaertim please address the linting error

File: packages/documentation/.storybook/styles/components/typo.scss (hunk 1 above)
Comment (schaertim) [reply packages/documentation/.storybook/styles/components/typo.scss:33]: Right now the post- prefix is optional. By removing it do you mean just removing it from the method calls using it right now?

**Review by gfellerph — APPROVED** [review]

File: packages/documentation/.storybook/styles/components/typo.scss (hunk 1 above)
Comment (oliverschuerch) [reply packages/documentation/.storybook/styles/components/typo.scss]: @alizedebray This has been implemented as you suggested it now 😉

**Pull request merged**